src/2_add_cots_priority.jl

```diff
@@ -0,0 +1,51 @@
+using
+    CSV,
+    Dates,
+    DataFrames
+
+using
+    GLMakie,
+    GeoMakie
+
+using
+    Statistics,
+    Bootstrap
+
+import GeoDataFrames as GDF
+import GeoFormatTypes as GFT
+import ArchGDAL as AG
+
+include("common.jl")
+
+# Load required data.
+RRAP_lookup = GDF.read(joinpath(OUTPUT_DIR,find_latest_file(OUTPUT_DIR)))
+cots_priority = CSV.read(joinpath(DATA_DIR, "CoCoNet_CoTS_control_reefs_2024.csv"), DataFrame, missingstring="NA")
+
+# Format data.
+rename!(cots_priority, Dict("LAT" =>  :cots_LAT, "LON" => :cots_LON,
+    "ReefID" => :GBRMPA_ID, "Priority" => :priority))
+cots_priority = cots_priority[:,[:GBRMPA_ID,:cots_LON,:cots_LAT,:priority]]
+
+# Adding cots priority into RRAP_lookup.
+RRAP_lookup = leftjoin(RRAP_lookup, cots_priority, on=:GBRMPA_ID, order = :left)
```

Comment:
No spaces in arguments.

```suggestion
RRAP_lookup = leftjoin(RRAP_lookup, cots_priority, on=:GBRMPA_ID, order=:left)
```